imports/ui/components/profile/profile_page.js

```diff
@@ -21,6 +22,8 @@ Template.profilePage.onCreated(function() {
         }
       }
     });
+
+    self.subscribe('allPosts');
```

Comment:
We should not be requesting every post in the database here. This would lead to massive downloads of data. Could you add a parameter to limit it to the user, and set the `fields` to only return the necessary info ie. timestamp, title, username?